python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -304,9 +307,13 @@ def evaluate(self, *, cache: MutableMapping[int, DataFrame]) -> DataFrame:
                     comment=comment,
                     decimal=decimal,
                     dtypes=self.schema,
-                    nrows=nrows,
+                    nrows=n_rows,
```

Comment:
Someone needs to check this works correctly with the `scan_csv` `skip_rows` argument.

Reply:
If not, feel free to push to this branch with tests (@lithomas1, perhaps)